### PR TITLE
Update sandman_uk.ts

### PR DESCRIPTION
**Pull request by SuperMaxusa**

### CODE REVIEW

File: SandboxiePlus/SandMan/sandman_uk.ts [hunk 1]
```diff
@@ -4182,7 +4140,6 @@ Please note that this values are currently user specific and saved globally for
     <message>
         <location filename="Forms/SettingsWindow.ui" line="619"/>
         <source>Only Administrator user accounts can use Pause Forced Programs Rules command</source>
-        <oldsource>Only Administrator user accounts can use Disable Forced Programs command</oldsource>
         <translation type="unfinished">Тільки адміністратор може використовувати команду &quot;Вимкнути примусові програми&quot;</translation>
```
Comment (isaak654) [inline SandboxiePlus/SandMan/sandman_uk.ts:4143]: The translated Rules word is missing

File: SandboxiePlus/SandMan/sandman_uk.ts [hunk 2]
```diff
@@ -2382,7 +2349,6 @@ No will choose: %2</source>
     <message>
         <location filename="Views/SbieView.cpp" line="946"/>
         <source>Do you really want to remove the selected sandbox(es)?&lt;br /&gt;&lt;br /&gt;Warning: The box content will also be deleted!</source>
-        <oldsource>Do you really want to remove the selected sandbox(es)?</oldsource>
         <translation type="unfinished">Ви дійсно хочете видалити цю(ці) пісочницю(і)?</translation>
```
Comment (isaak654) [inline SandboxiePlus/SandMan/sandman_uk.ts:2352]: The translation here is incomplete, you need to translate `Warning: The box content will also be deleted!`

File: SandboxiePlus/SandMan/sandman_uk.ts [hunk 3]
```diff
@@ -1344,47 +1332,42 @@ No will choose: %2</source>
...
         <source>The supporter certificate is not valid for this build, please get an updated certificate</source>
-        <translation type="unfinished"></translation>
+        <translation>Цей сертифікат спонсора не є дійсним для цієї збірки, будь ласка, оновіть сертифікат</translation>
     </message>
     <message>
         <location filename="SandMan.cpp" line="2364"/>
         <source>The supporter certificate has expired%1, please get an updated certificate</source>
-        <oldsource>The supporter certificate is expired %1 days ago, please get an updated certificate</oldsource>
-        <translation type="unfinished"></translation>
+        <translation>Цей сертифікат спонсора був вичерпаний%1, будь ласка, отримайте новий сертифікат</translation>
```
Comment (isaak654) [inline SandboxiePlus/SandMan/sandman_uk.ts:1345]: The %1 value is supposed to stay attached by design, don't change it in future revisions.

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Comment by SuperMaxusa** [comment]
Thanks for your reviews. I fixed this now.

**Comment by isaak654** [comment]
There are still a number of formatting issues to fix.

1) No `%1 (%2)` markers in the translated line here:
https://github.com/sandboxie-plus/Sandboxie/blob/c3f87a95965f5be0353aea21c421e4c13c8a7648/SandboxiePlus/SandMan/sandman_uk.ts#L735-L736

2) No %1 marker in the translated line here:
https://github.com/sandboxie-plus/Sandboxie/blob/c3f87a95965f5be0353aea21c421e4c13c8a7648/SandboxiePlus/SandMan/sandman_uk.ts#L740-L743

3) No period here:
https://github.com/sandboxie-plus/Sandboxie/blob/c3f87a95965f5be0353aea21c421e4c13c8a7648/SandboxiePlus/SandMan/sandman_uk.ts#L761-L762

4) No colon needed here (because it's a button):
https://github.com/sandboxie-plus/Sandboxie/blob/c3f87a95965f5be0353aea21c421e4c13c8a7648/SandboxiePlus/SandMan/sandman_uk.ts#L4361-L4362

I can't list them all, however I suggest you to load the TS file in the Linguist tool (linguist.exe).
See [Localization notes and tips](https://github.com/sandboxie-plus/Sandboxie/discussions/1123#discussioncomment-1203489) if you need further details.

Once loaded the TS file, the tool will indicate you the entries to fix (they are all marked in yellow). It will also help you to prevent any type of formatting issue in future.

**Pull request closed without merging**